jekyll/_cci2/insights.md

```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
 
 ![header]({{ site.baseurl }}/assets/img/docs/insights-current-build.png)
 
+
 - **Build Status:** The Insights dashboard shows the last 50 builds for your default branch. Click a branch in the top right corner to access over 100 build/job statuses for the selected branch.
 
 - **Build Performance:** The Build Performance graph aggregates your build/job data for a particular day and plots the median for that day going back as far as 90 days. Monitor the performance of your repo by clicking a particular branch.
 
 ## See Also
 
 Refer to the [Collecting Test Metadata]({{ site.baseurl }}/2.0/collect-test-data/) document for instructions to configure insights into your most failed tests.
+
+## Sumo Logic Integration
+
+Sumo Logic users may now track and visualize analytical data  across all of their jobs on CircleCI by utilizing the new Sumo Logic Orb and CircleCI App for Sumo Logic app integration on Sumo Logic's partner integrations site.
+
+
+### The CircleCI Dashboard for Sumo Logic
+
+![header]({{ site.baseurl }}/assets/img/docs/CircleCI_SumoLogic_Dashboard.png)
```

Comment:
this needs a description for 508 compliance (images must have descriptive alt text).